### PR TITLE
Adding checks for None for flows is_filtered flag

### PR DESCRIPTION
**Pull request by Sonamsinhaarkin**
Epic:
https://jira.eng.vmware.com/browse/CYG-85319

Tracking Jira:
https://jira.eng.vmware.com/browse/CYG-93282

Summary:
Adding checks for None for flows is_filtered flag

Testing screenshots:
Screenshot 2023-12-19 at 10 30 53 AM

### CODE REVIEW

File: databus_client/queues/entity_queues/databus_flows_queue.py [hunk 1]
```diff
@@ -67,9 +67,14 @@ def start_processing_data(self):
                     for flow in flow_data:
                         try:
                             message = dict()
-                            self.append_key_val_in_dict(message,
-                                                    ["id", "type", "specversion", "source", "message_group", "token", "is_filtered"],
-                                                    entry)
+                            if is_filtered is not None:
```
Comment (prasadinglearkin) [inline databus_client/queues/entity_queues/databus_flows_queue.py:70]: if is_filtered
only this shall work

File: databus_client/endpoint/client_endpoint.py [hunk 2]
```diff
@@ -97,7 +97,7 @@ def databus_switchports_metrics():
 
 @app.route('/' + DatabusMessageGroup.FLOWS.value, methods=['GET', 'POST', 'DELETE', 'COPY'])
 def databus_flows():
-    return databus_queue_processor(request=request, message_group=DatabusMessageGroup.FLOWS.value)
+    return databus_queue_processor(request=request, message_group=DatabusMessageGroup.FLOWS.value, is_filtered=False)
```
Comment (prasadinglearkin) [inline databus_client/endpoint/client_endpoint.py:100]: Not needed

File: databus_client/endpoint/client_endpoint.py [hunk 3]
```diff
@@ -316,7 +316,7 @@ def do_post(queue_processor=None, message_group=None, is_filtered=None):
     info_dict = {"message_group": message_group}
     try:
         data = request.json
-        if is_filtered:
+        if is_filtered is not None:
```
Comment (prasadinglearkin) [inline databus_client/endpoint/client_endpoint.py:319]: Not needed

**Review by prasadinglearkin — CHANGES_REQUESTED** [review]

File: databus_client/queues/entity_queues/databus_flows_queue.py (hunk 1 above)
Comment (Sonamsinhaarkin) [reply databus_client/queues/entity_queues/databus_flows_queue.py]: changes done

File: databus_client/endpoint/client_endpoint.py (hunk 2 above)
Comment (Sonamsinhaarkin) [reply databus_client/endpoint/client_endpoint.py]: changes done

File: databus_client/endpoint/client_endpoint.py (hunk 3 above)
Comment (Sonamsinhaarkin) [reply databus_client/endpoint/client_endpoint.py]: changes done

**Review by prasadinglearkin — APPROVED** [review]

**Pull request merged**